english/find-exit/into-mountains/upwards/further-upwards/towards-saddle/rush/inspect/burn_aspect/burn_aspect.md

```diff
@@ -0,0 +1,19 @@
+You gather the materials for the ritual as evening draws. You find a small abandoned cave and light the fire.
+After a period of ceremonial meditation, you open your eyes and cast the small Aspect into the flames. Almost immediately a billowing black smoke erupts from the fire and you are thrown into a deep trance.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.